Woodoku/src/entity/woodoku_board.py

```diff
@@ -159,28 +160,26 @@ def add_shape(self, shape: WoodokuShape, x: int, y: int) -> None:
         self.__representation.add_blocks(shape_blocks)
 
         # determine groups and clear the groups
-        group_info, group_blocks = self.__find_groups()
-        self.__scoreAgent.calculate_winning(group_info)
+        groups, group_blocks = self.__find_groups()
+        self.__scoreAgent.calculate_winning(len(shape), groups)
         self.__representation.remove_blocks(group_blocks)
 
     def get_score(self) -> int:
         return self.__scoreAgent.get_score()
 
-    def __find_groups(self) -> Tuple[Dict[str, int], Set[Tuple[int, int]]]:
+    def __find_groups(self) -> Tuple[int, Set[Tuple[int, int]]]:
```

Comment:
@YANLING-H Changes in test cases might be required.

Reply:
#32 @YANLING-H, there perhaps need some change